das/daser.go

```diff
@@ -51,10 +50,6 @@ func (d *DASer) Start(context.Context) error {
 
 // Stop stops sampling.
 func (d *DASer) Stop(ctx context.Context) error {
-	if d.cancel == nil {
-		return fmt.Errorf("da: DASer already stopped")
-	}
-
```

Comment:
Instead, If it is already stopped, there will be a panic. IMO, panic in this case is better as it tells that we are using code incorrectly. Meaning we are Starting once, but stopping twice. This is only relevant for Stop. Maybe this will change once we have restartable stuff during runtime.